src/commands/checkModel.ts

```diff
@@ -316,6 +351,71 @@ async function checkModelExists(cfgPath: string, warn = true): Promise<boolean>
     return cfgExists;
 }
 
+async function readExtendsModules(modelPath: string): Promise<string[] | undefined> {
```

Comment:
Using SANY instead of regular expressions would be simpler and more robust.

Reply:
dealt with in the last commit